samples/msgext-search-sso-config/nodejs/appPackage/manifest.json

```diff
@@ -94,8 +94,9 @@
           "type": "link",
           "value": {
             "domains": [
-              "*.botframework.com"
-            ]
+              "*.botframework.com",
+            ],
+			"supportsAnonymizedPayloads": true
```

Comment:
Please check and correct the indentation.

Reply:
Fixed the indentation